gradle/libs.versions.toml

```diff
@@ -142,8 +142,8 @@ hive3-service = { module = "org.apache.hive:hive-service", version.ref = "hive3"
 httpcomponents-httpclient5 = { module = "org.apache.httpcomponents.client5:httpclient5", version.ref = "httpcomponents-httpclient5" }
 immutables-value = { module = "org.immutables:value", version.ref = "immutables-value" }
 jackson-bom = { module = "com.fasterxml.jackson:jackson-bom", version.ref = "jackson-bom" }
-jackson-core = { module = "com.fasterxml.jackson:jackson-core", version.ref = "jackson-bom" }
-jackson-databind = { module = "com.fasterxml.jackson:jackson-databind", version.ref = "jackson-bom" }
+jackson-core = { module = "com.fasterxml.jackson.core:jackson-core", version.ref = "jackson-bom" }
```

Comment:
can you rebase this PR please? I've fixed this in a642a9350e06dc244ec167372985564372896850

Reply:
should be irrelative, anyway, let me rebase it